ncm-systemd/src/main/perl/Systemd/Service/Unit.pm

```diff
@@ -462,6 +459,14 @@ sub configured_units
         $units{$detail->{name}} = $detail;
     }
 
+    if (@tryrestart) {
+        # It is ok to do the conditional restart here. This does not change state.
```

Comment:
What does this comment mean? Why might it not be OK to do the conditional restart here? What is the "state" that is not changed -- akin to keeps_state=>1 or because the restart should keep the service in the same running/not-running state as before? (In which case what about the case where stop succeeds but start fails)?